### ISSUE TITLE
Doesn't support 🍆

### ISSUE THREAD

**Issue by nv-vn** [installgen2/libspurdo#3]
libspurdo has no replacement for the eggplant emoji (or emojis in general). This is highly mysoginistic because libspurdo does not filter out emojis which may be offensive to womyn*, as well as transfolk, otherkin, and people with PTSD.

**Comment by installgen2** [comment]
I agree, I am highly disappointed with the way that women are treated in our online coding communities, I will make an effort to improve my application to facilitate the needs of them and make GitHub a safer place for all women. I will attempt to have a fix for this ready by tomorrow, and will also add fixes for other emojis that could possibly be considered offensive. I hope one day we can purge this terrible planet of the cancer that makes it this way, the problem we really isn't global warming or the usage of fossil fuels, but the extreme amount of phallic objects that women have to encounter every day.

**Comment by nv-vn** [comment]
:+1:

**Comment by nv-vn** [comment]
:+1:

**Comment by nv-vn** [comment]
:100:
